src/reducers/index.js

```diff
@@ -63,12 +74,15 @@ function getInitialState() {
       subRowHeightGetter: () => 0,
     },
     columnSettings: {
+      defaultColumnWidth: 100,
```

Comment:
This is the initial column width that FDT assumes for every column.
Once a column is deemed to be in the visible viewport, FDT gets the actual width from the user.